controllers/login.js

```diff
@@ -11,6 +11,7 @@ const authHelper = require('../helpers/authentication');
 const userConsentVersions = require('../helpers/consentVersions');
 
 const logger = require('../helpers/logger');
+const { LOGIN_BLOCK_TIME } = require('../config/global');
```

Comment:
you didn't add this, but there is no default defined. the configuration should be used instead of globals.

Reply:
ok, resolved